packages/orchestration/src/examples/sendAnywhere.contract.js

```diff
@@ -134,7 +134,7 @@ export const start = async (zcf, privateArgs, baggage) => {
        */
       async addChain(chainInfo, connectionInfo) {
         const chainKey = `${chainInfo.chainId}-${(nonce += 1n)}`;
-        const agoricChainInfo = await chainHub.getChainInfo('agoric');
+        const agoricChainInfo = await V.when(chainHub.getChainInfo('agoric'));
```

Comment:
Nice. Consider an inline comment explaining why using `V.when()` is safe here, since this is a reference contract.